sqlglot/dataframe/sql/functions.py

```diff
@@ -908,6 +908,10 @@ def hex(col: ColumnOrName) -> Column:
     return Column.invoke_expression_over_column(col, expression.Hex)
 
 
+def upperhex(col: ColumnOrName) -> Column:
+    return Column.invoke_expression_over_column(col, expression.UpperHex)
```

Comment:
why was this added?

Reply:
@georgesittas hmm as mentioned as below, i made `def test_upperhex(self):` to represent the logic of the old `def test_hex(self):`

Reply:
Functions defined in this file are supposed to match pyspark's dataframe API, so not sure if it makes sense to define `upperhex` here. If you _needed_ to add this, it's probably wrong, cc @eakmanrq can you verify?

Reply:
that's right, these functions are only used for pyspark's dataframe api

Reply:
I have removed changes in this file and test also.